src/OAuth/Client.php

```diff
@@ -94,6 +99,11 @@ public function getRedirectUri(): array
         return $this->redirectUri;
     }
 
+    public function setRedirectUri(array $redirectUri): void
```

Comment:
variadic function would be a nice way to protect non String values from entering the array.

Reply:
Based on the PropertyAccess component [documentation](https://symfony.com/doc/current/components/property_access.html#writing-to-array-properties), I'll probably refactor this to `addRedirectUri(string $uri)`/`removeRedirect(string $uri)` with Beberlei/Assert checks builtin.